Task 8/script.js

```diff
@@ -0,0 +1,30 @@
+/* ------------------------------ TASK 8 --------------------------------------------
+Sukurkite konstruktoriaus funkciją "Calculator" (naudokite ES5), kuri gebės sukurti objektus su 4 metodais:
+sum(a, b) - priima du skaičius ir grąžina jų sumą.
+subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
+multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
+division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
+------------------------------------------------------------------------------------ */
+
+
+class Calculator {
+    constructor(a, b) {
```

Comment:
I think you missread exercise, there is few things:

- This should be constructor function (ES5) it means it's not a class but older version function.
- Methods should accept argument themselves, it should not go into constructor.